js/tests/unit/util/scrollbar.spec.js

```diff
@@ -28,17 +28,19 @@ describe('ScrollBar', () => {
 
   afterAll(() => {
     fixtureEl.remove()
-    document.documentElement.style.overflowY = 'auto'
-    document.body.style.overflowY = 'auto'
   })
 
   afterEach(() => {
     clearFixture()
-    document.documentElement.removeAttribute('style')
+    document.documentElement.style.removeProperty('paddingRight')
```

Comment:
Maybe use `document.documentElement.removeAttribute('style')` here too to keep things in line with the other cleanups?